### ISSUE TITLE
relevanssi_post_date_throttle_join replaces Join instead of adding something

### ISSUE THREAD

**Issue by medonat** [msaari/relevanssi#55]
I probably found a bug in search.php:1980

It seems that the `relevanssi_post_date_throttle_join` function should add something to the join statement but instead replaces the join statement.

If I replace the assignment with an concatenating assignment operator our error is gone:
```
function relevanssi_post_date_throttle_join( $query_join ) {
	if ( 'post_date' === get_option( 'relevanssi_default_orderby' ) &&
		'on' === get_option( 'relevanssi_throttle', 'on' ) ) {
		global $wpdb;
		$query_join .= ', ' . $wpdb->posts . ' AS p';
	}
	return $query_join;
}
```

**Comment by msaari** [comment]
Yeah, that's an obvious error. Thanks for spotting it! I'll fix this in the next release.

**Comment by msaari** [comment]
Fixed in 4.22.1.
